assets/stylesheets/components/_paginator.scss

```diff
@@ -0,0 +1,40 @@
+.paginator {
```

Comment:
Based on our convention, the class name should be composed of at least two words https://nimblehq.co/compass/development/code-conventions/css/#class-names